client/src/charts/canada/CanadaD3Component.js

```diff
@@ -201,9 +201,9 @@ export class CanadaD3Component {
             position: "absolute",
             "border-radius": "5px",
             "text-align": "center",
-            "font-size": "10px",
+            "font-size": scale * 22 + "px",
```

Comment:
I think this should be written `${scale * 22}px`

Reply:
Noted and fixed  👍